### PR TITLE
Require Perl v5.30, Node v16

### PR DESCRIPTION
**Pull request by mwiencek**
https://blog.metabrainz.org/2021/03/17/schema-change-release-may-17-2021/

Note: This switches the underlying Ubuntu base image to focal.

### CODE REVIEW

**Review by reosarevok — APPROVED** [review]
This seems sensible. Please do check whether https://github.com/metabrainz/musicbrainz-server/pull/1714 / https://github.com/metabrainz/musicbrainz-server/pull/1736 can be reverted now and add an extra commit here if so :)

**Comment by mwiencek** [comment]
Done. Also bumped Node to v16.1.0, since that was released today.

**Review by reosarevok — APPROVED** [review]

File: docker/templates/macros.m4 [hunk 1]
```diff
@@ -23,9 +23,9 @@ copy_mb(``package.json yarn.lock ./'')
 RUN apt-key add /tmp/yarn_pubkey.txt && \
     rm /tmp/yarn_pubkey.txt && \
     echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
-    apt_install(``git python-minimal yarn'') && \
+    apt_install(``git python3-minimal yarn'') && \
```
Comment (yvanzo) [inline docker/templates/macros.m4:26]: That has been added in commit b68b7b853e8abd82f41a1c0eabc09a3ef48afb32. Is it still required nowadays?

**Review by yvanzo — COMMENTED** [review]
@brainzbot retest this please

File: docker/templates/macros.m4 (hunk 1 above)
Comment (mwiencek) [reply docker/templates/macros.m4:26]: Yes, nodejs still depends on it: https://deb.nodesource.com/node_16.x/dists/focal/main/binary-amd64/Packages

**Comment by mwiencek** [comment]
@brainzbot test this please

**Comment by mwiencek** [comment]
@brainzbot retest this please

**Pull request merged**